src/Duo.Clients.Wpf/Presentation/ManutenzioneTrattamentoViewModel.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Duo.Domain.ViewModels.Trattamenti;
+using System;
+using System.Text;
+using Topics.Radical.ComponentModel.Messaging;
+using Topics.Radical.Windows.Presentation;
+using Topics.Radical.Windows.Presentation.Messaging;
+
+namespace Duo.Clients.Wpf.Presentation
+{
+    class ManutenzioneTrattamentoViewModel : AbstractViewModel
+    {
+        readonly IMessageBroker broker;
+
+        public TrattamentoView Trattamento
```

Comment:
Vedi sopra... in effetti in ottica CQRS non si dovrebbero usare View per binding e modifica di logica. 
Vedi se la correzione va bene.